app.js

```diff
@@ -1,117 +1,186 @@
 const express = require("express");
+const db = require("./databaseConnect");
 const app = express();
 const port = 3001;
+const jwt = require("jsonwebtoken");
+const authMiddleware = require("./authMiddleware");
+
+db.init();
 
 var hash = require("pbkdf2-password")();
 
 var cors = require("cors");
+const { json } = require("express");
```

Comment:
why you have extracted this josn? it's not used anywhere

Reply:
done

Reply:
I don't see another PR opened into the server-database branch as requested